subset/security/tlsfaux/generate_certs.py

```diff
@@ -0,0 +1,16 @@
+import subprocess
+
+filepath = 'certs/'
+
+command_key = 'openssl req -nodes -newkey rsa:2048 -keyout '+filepath+'server.key -out '+filepath+'server.csr -subj "/C=GB/ST=London/L=KingsX/O=ExcelRedstone/OU=Software/CN=127.0.0.1"'
```

Comment:
use [python String formatting](https://www.learnpython.org/en/String_Formatting) here -- makes it a lot easier to read

Reply:
implemented python string formatting